tests/unit/lms/views/api/exceptions_test.py

```diff
@@ -232,6 +247,9 @@ def pyramid_request(self, pyramid_request):
         # exception that was raised by the original view:
         # https://docs.pylonsproject.org/projects/pyramid/en/latest/api/request.html#pyramid.request.Request.exception
         pyramid_request.exception = ValueError()
+
+        pyramid_request.matched_route = Mock(spec_set=["name"])
```

Comment:
Pyramid's `DummyRequest` doesn't have a `matched_route` attribute, even though real requests do. I don't think there's any better way to mock a Pyramid `Route` object than this: it's not `create_autospec()`-friendly

Reply:
Yeah it's a bit too dummy, but this works.